.github/workflows/ci.yml

```diff
@@ -46,6 +46,6 @@ jobs:
         PG_PORT: ${{ job.services.postgres.ports['5432'] }}
         MYSQL_DATABASE: mysql
         MYSQL_HOST: localhost
-        MYSQL_PORT: ${{ job.services.mysql.ports['5432'] }}
+        MYSQL_PORT: ${{ job.services.mysql.ports['3306'] }}
```

Comment:
This probably never worked? It probably interpolated into the empty string, and the tests used the default port which is 3306